java/java.hints/test/unit/src/org/netbeans/modules/java/hints/suggestions/ImplementMethodsTest.java

```diff
@@ -44,7 +44,7 @@ public void testSimple1() throws Exception {
                 .assertOutput("package test;\n" +
                               "public abstract class Test implements Runnable {\n" +
                               "    public void run() {\n" +
-                              "        throw new UnsupportedOperationException(\"Not supported yet.\"); //To change body of generated methods, choose Tools | Templates.\n" +
+                              "        throw new UnsupportedOperationException(\"Not supported yet.\"); // Generated from nbfs://nbhost/SystemFileSystem/Templates/Classes/Code/GeneratedMethodBody\n" +
```

Comment:
There doesn't seem to be any way to treat the comment as `.*` without specifying the exact content, right @jlahoda?

Reply:
Let's assume there is no better way and all tests had to be modified in ba90ee1